spec/models/course/announcement_spec.rb

```diff
@@ -6,70 +6,60 @@
 
   let!(:instance) { create(:instance) }
   with_tenant(:instance) do
-    context 'when announcement is created' do
-      subject { Course::Announcement.new }
+    let!(:user) { create(:user) }
+    let(:course) { create(:course) }
 
-      it { is_expected.not_to be_sticky }
-    end
-
-    context 'when title is not present' do
-      subject { build(:course_announcement, title: '') }
+    describe 'create an announcement' do
+      context 'when announcement is created' do
+        subject { Course::Announcement.new }
 
-      it { is_expected.not_to be_valid }
-    end
+        it { is_expected.not_to be_sticky }
```

Comment:
@lowjoel I'm not sure if I get you, or do you mean the specs for `unread status` ?